src/models/user.py

```diff
@@ -30,11 +31,12 @@ def fields(self):
             'date_registered' : (b.is_date_string,),
             'last_sign_in' : (b.is_date_string,),
             'courses' : (b.is_list,),
-            'departments' : (b.is_list,),
+            'departments' : (),
             'created_surveys' : (b.is_list,),
             'unanswered_surveys' : (b.is_list,),
             'answered_surveys' : (b.is_list,),
             'answers' : (b.is_list,),
+            'primary_affiliation' : (),
```

Comment:
You're right, I just had to leave 'is_list' out because it wasn't working at the time. It should be fine now.